src/RoomsSDKAdapter.js

```diff
@@ -0,0 +1,35 @@
+import {Observable} from 'rxjs';
+import {RoomsAdapter} from '@webex/components';
+
+export default class RoomsSDKAdapter extends RoomsAdapter {
+  constructor(sdk) {
+    super(sdk);
+
+    this.sdk = sdk;
+  }
+
+  /**
+   * Returns an observable that emits room data of the given ID.
+   *
+   * @param {string} ID ID of room to get
+   * @returns {Observable.<Room>}
+   * @memberof RoomsSDKAdapter
+   */
+  getRoom(ID) {
+    return Observable.create(async (observer) => {
+      try {
+        const room = await this.sdk.rooms.get(ID);
```

Comment:
How would we handle room updates, such as updating the title of the room?

Reply:
The sdk does not provide live updates through the public api.

Reply:
How would we get live updates otherwise?

Reply:
Nevermind, looks like JP added this feature to the rooms plugin a couple months ago. Will update the PR.